core/modules/layout/plugins/renderers/layout_renderer_standard.inc

```diff
@@ -362,6 +362,11 @@ class LayoutRendererStandard {
         }
       }
     }
+    if (!empty($this->layout_info['libraries'])) {
+      foreach ($this->layout_info['libraries'] as $library) {
+        $this->addLibrary('layout', $library);
```

Comment:
This initially made me a bit nervous that "layout" is hard-coded. However it would still be possible for other modules to provide a library by using `hook_library_info_alter()`, so this isn't quite so limiting as it may seem at first. We may want to revisit this in the future, especially as themes should have this ability to use libraries as well.